src/routing/comm.rs

```diff
@@ -105,7 +105,12 @@ impl Comm {
         msg: Bytes,
     ) -> Result<(), SendError> {
         if let Some(conn) = self.endpoint.get_connection(client) {
-            if conn.send_uni(msg.clone()).await.is_ok() {
+            if let Err(err) = conn.send_uni(msg.clone()).await {
```

Comment:
We are still swallowing an error here, should we? I imagine as this returns a Result we need to return the error as a result here and not swallow it (prob why we have the issue in the first place)

Reply:
This is to be addressed within the PR https://github.com/maidsafe/sn_routing/pull/2286